api/src/main/resources/sql/procedures/create_functions.sql

```diff
@@ -172,6 +172,36 @@ RETURN dosId_out;
 
 END;
 #
+
+/*
+The following will return the primary patient identifier defined in the system's global properties for a given patient id 
```

Comment:
change to "defined in the system's metadata mappings"... sorry I led you astray with the GP solution but that has been deprecated.